### ISSUE TITLE
Design entspricht nicht mehr dem des Bundesverbands

### ISSUE THREAD

**Issue by larsschellhas** [kre8tiv/urwahl3000#72]
Ich mag Urwahl3000 sehr! Aber der Satz:

> Das Layout orientiert sich am Layout von gruene.de. 

trifft leider einfach nicht mehr zu, wie mir inzwischen auch einige Kreisverbände gesagt haben, die ich Webseiten-technisch betreue.

Habt ihr hier Pläne, hinsichtlich eines Updates/Relaunchs?

**Comment by modulbuero** [comment]
Hallo @larsschellhas ,

das ist völlig richtig – auch wenn gruene.de damit derzeit im GRÜNEN Internet eine ziemliche Ausnahmestellung einnimmt. Trotzdem sind wir natürlich schon an den Überlegungen, ob und wenn ja wie wir hier eine Angleichung vornehmen, und führen dazu auch Gespräche mit dem Bundesverband. Mehr kann ich dir leider zum aktuellen Zeitpunkt noch nicht sagen als das: wir haben es auf dem Schirm und durchdenken es :-) 

Viele Grüße,
Christian

**Comment by larsschellhas** [comment]
Hey Christian,
alles klar, danke für die Info!
Vor dem Hintergrund würde ich das Issue aber gerne offen lassen, bis es hierzu Neuigkeiten gibt :)
Viele Grüße
Lars

**Comment by modulbuero** [comment]
Klar, kein Problem. Ich kann nur nicht versprechen, dass ich mich daran erinnern werde, mich hier zurück zu melden :-D

**Comment by larsschellhas** [comment]
Alles klar :D 
Im Zweifelsfall frage ich in ein paar Wochen/Monaten nochmal nach^^

**Comment by larsschellhas** [comment]
Gibt's schon was neues in der Diskussion? :D 
Sonst melde ich mich in nem Monat nochmal 😉

**Comment by modulbuero** [comment]
@larsschellhas Nein, wir sind weiterhin dran.

**Comment by modulbuero** [comment]
Eine kurze Info hier: Die Gespräche dazu laufen weiter, wir gehen im Moment aber nicht von einer Umsetzung vor dem Frühjahr 2021 aus.

**Comment by thomasgoetzelmann** [comment]
Gibt es mittlerweile Neuigkeiten zu einem etwaigen neuen Theme/Update?

**Comment by mnbvghw** [comment]
[Hier](https://github.com/codeispoetry/sunflower) wird gerade ein neues Theme erarbeitet mit einer leichten Umstiegsmöglichkeit von Urwahl3000

**Comment by larsschellhas** [comment]
Jetzt haben wir Sommer 2021 und es gibt noch keine Neuigkeiten. Ich schätze mal, Sunflower ist dann wohl der Nachfolger von Urwahl3000 oder kommt doch noch was @modulbuero ?
